articles/introduction-to-sequalize-orm-for-nodejs/index.md

```diff
@@ -0,0 +1,215 @@
+### Introduction to Sequelize ORM for Node.js
+![hero image](hero.jpg)
+
+### Introduction
+Connecting an Object-relational Mapper to a Node.js application is not an easy task for most developers since there are no better elaborate resources and documentation about it. If you're one, then this article is for you. It also covers SQL basics so beginners in databases will find it insightful. We will not dive though into how Sequelize ORM integrates with a Nodejs application. Let us save that for a future tutorial.
```

Comment:
Before we directly jump into ORM for Node.js, we could start with what is `Sequelization`, then what is `ORM`, and what is the use of connecting ORM with Node.js

So, make sure to elaborate your intro.